### PR TITLE
New getty exclusions

### PR DESCRIPTION
**Pull request by jamesgorrie**
New list from Jo B.

## Free count:
`before: 1645942`
`after: 1651972`

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala [hunk 1]
```diff
@@ -5,90 +5,35 @@ object UsageRightsConfig {
...
-    "Ulrich Baumgarten",
-    "Universal Images",
-    "World Kabbadi League",
-    "ullstein bild"
+    "Sports Illustrated Classic",
+    "Terry O'Neill",
+    "The Asahi Shimbun Premium Archive",
+    "The LIFE images collection",
+    "The LIFE picture collection Editorial",
+    "The LIFE premium collection"
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: capitalisation has changed. I think capitalisation may matter -- is it worth checking for matches?

**Comment by theefer** [comment]
Comment about capitalisation, else :+1:

**Comment by jamesgorrie** [comment]
> capitalisation

You were right, nice catch, updated and checked the rest from the list.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala [hunk 2]
```diff
@@ -5,90 +5,35 @@ object UsageRightsConfig {
...
-    "Her Og Nu",
-    "Hero Images",
-    "Hulton Royals Collection",
-    "IAAF World Athletics",
-    "ICC",
-    "Imperial War Museums",
-    "Interact Images",
+    "Fashion Window",
+    "Gamma-Legend",
+    "Gamma-Rapho",
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: This has also had a hyphen materialise. Can we check if one is the canonical one and which one?

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala [hunk 3]
```diff
@@ -5,90 +5,35 @@ object UsageRightsConfig {
...
-    "Caiaimage",
-    "Carnegie Museum Art",
+    // TODO: Remove once we are on the new UsageRights model
+    // This is not excluded as we have a deal with Barcroft,
+    // and there are only a tiny amount from Getty
+    // "Barcroft Media",
     "Catwalking",
-    "Champions Hockey League",
-    "City-Press",
+    "Contour by Getty Images",
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: Was "Contour" before.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala [hunk 4]
```diff
@@ -5,90 +5,35 @@ object UsageRightsConfig {
...
+    "Fashion Window",
+    "Gamma-Legend",
+    "Gamma-Rapho",
     "International Center of Photography",
-    "Kommersant",
     "Lichfield Archive",
-    "LightRocket",
-    "MLB Platinum",
-    "Mark Leech Sports Photography",
+    "MLB Major League Baseball Platinum",
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: Was "MLP Platinum" before.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala [hunk 5]
```diff
@@ -5,90 +5,35 @@ object UsageRightsConfig {
...
-    "Toronto Star",
-    "UEFA.com",
-    "UK Press",
-    "Ulrich Baumgarten",
-    "Universal Images",
-    "World Kabbadi League",
-    "ullstein bild"
+    "Sports Illustrated Classic",
+    "Terry O'Neill",
+    "The Asahi Shimbun Premium Archive",
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: These two also had other syntax before, can we check?

**Comment by theefer** [comment]
Sorry but there are more discrepancies in the previous vs new collection names that I think we should check.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala (hunk 3 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: `"Contour by Getty Images"` is correct.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala (hunk 2 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: The hyphen is correct for `"Gamma-Rapho"` - Legend doesn't have any, but assuming it's the same.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala (hunk 4 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: Neither match - but assuming as the new ones seem to be correct, this must be correct.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala (hunk 5 above)
Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: Which two?

**Comment by theefer** [comment]
Great thanks for checking. If we really wanted to check we could probably download images from the Getty archive in the respective collections, but given there are no images now hopefully they're not collections under heavy use.

File: common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala (hunk 5 above)
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: Terry O'Neill (curly apostrophe vs straight single quote), and "The Asahi Shimbun Premium [Archive]".

Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: The single quote must have been a copy paste balls up as it's the apostrophe.
Checking with Jo on the [Archive].

Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: Only ones in the archive are excluded.

Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: OK great! The master version has the straight single quote for Terry O'Neill, are you saying it should be the curly one?

Comment (jamesgorrie) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: It's meant to be the straight on (non match the curly, so we must have been getting it wrong).
It is now correct.

Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/config/UsageRightsConfig.scala]: OK :sunny:

**Pull request merged**